src/ui.form-label/ULabel.vue

```diff
@@ -88,13 +88,16 @@ const { wrapperAttrs, contentAttrs, labelAttrs, descriptionAttrs } = useUI<Confi
 
     <!-- `v-bind` isn't assigned, because the div is system -->
     <div v-if="label || error || description">
-      <label
-        v-if="label"
-        :for="props.for"
-        v-bind="labelAttrs"
-        :data-test="`${dataTest}-label`"
-        v-text="label"
-      />
+      <!-- @slot Use it to add something instead of label. -->
+      <slot name="label" :for="props.for">
+        <label
+          v-if="label"
+          :for="props.for"
+          v-bind="labelAttrs"
+          :data-test="`${dataTest}-label`"
+          v-text="label"
+        />
```

Comment:
no, need to add slot inside of the label tag to keep label functionality and allow to place inside some custom component if needs.

```vue
<label
  v-if="label"
  :for="props.for"
  v-bind="labelAttrs"
  :data-test="`${dataTest}-label`"
>
  <!-- @slot Use it to add something instead of label. + !!!define label slot prop-->
  <slot name="label" :label="label">
    {{ label }}
  </slot>
</label>
```